src/main/resources/templates/400.html

```diff
@@ -0,0 +1,17 @@
+<page th:replace="~{fragments/user-layout :: page(~{::title},~{::content})}">
+    <title>Bad request</title>
+
+    <content>
+        <div class="container">
+            <div class="grid-row">
+                <div class="column-two-thirds">
+                    <h1 class="heading-xlarge">Sorry, there is a problem with the service</h1>
```

Comment:
The message on the other services is "Sorry, there is a problem with this service" (replace "the" with "this")

Reply:
Review comments applied.